src/js/entry.js

```diff
@@ -208,3 +219,21 @@ export function rootApp() {
         );
     }
 }
+
+export function noAccess() {
+    const { store } = spinUpStore();
+    const app = location.pathname.split('/').pop();
+    if (app === 'insights') {
+        const userEntitlements = window.insights.chrome.auth.getUserEntitlements();
+        const grantAccess = userEntitlements.insights.is_entitled;
+        if (!grantAccess) {
```

Comment:
This isn't going to scale very well when we get more apps. Can we abstract this check to another function somewhere? That way, we don't have to do a big rework when Ansible's Automation Hub, for example, needs some kind of check.

Reply:
I went for an object here. Let me know if this doesn't work!